package.xml

```diff
@@ -53,7 +53,7 @@
   <run_depend>tf2_eigen</run_depend>
   <run_depend>tf2_geometry_msgs</run_depend>
 
-  <test_depend>moveit_resources</test_depend>
+  <test_depend>moveit_resources_panda_moveit_config</test_depend>
```

Comment:
Should we switch to native panda_moveit_config here? There is no need to use moveit_resources here for testing, is it?

Reply:
There is one tutorial on creating tests that uses our panda model from resources ([see here](https://github.com/ros-planning/moveit_tutorials/blob/e49e1b0ac4dcb646bcc9295bfe04261d664aa8f8/doc/tests/test/tests.cpp#L72)).
The rest is supposed to use the native robot.